expected-consensus.md

```diff
@@ -292,12 +292,12 @@ See the [Faults spec](./faults.md) for implementation details.
 
 Due to the existence of potential forks, a miner can try to unduly influence protocol fairness. This means they may choose to disregard the protocol in order to gain an advantage over the power they should normally get from their storage on the network. A miner should be slashed if they are provably deviating from the honest protocol.
 
-This happens in the following instances: 
+This is detectable when a miner submits two blocks that satisfy either of the following "slashing conditions":
 
-- A miner submits two blocks for the same round.
-- A miner mines atop a TipSet that should have contained their own block (same height, same parents) but does not.
-  - A miner may omit their own block if they find a winning Ticket in the parent TipSet though they themselves have submitted a smaller (losing) ticket. While it cannot be proven that block omission from a TipSet is malicious (i.e. network latency could simply mean the miner did not receive a particular block), for obvious reasons, a miner must be are aware of the block they previously mined.
-- Also: https://github.com/filecoin-project/consensus/issues/45#issuecomment-468035739
+(1) one block contains at least one ticket in its ticket array generated at the same round as one of the tickets in the other block's ticket array.
+(2) one block's ancestor is a TipSet that could have validly included the other block according to TipSet validity rules, however that ancestor of the first block does not include the other block.
```

Comment:
Just want to make sure that by 'ancestor' we mean explicitly *direct* ancestor. If miner A mines on top of X to produce Y, and another miner mines on top of X to produce Z, then another miner mines on top of Z to produce W, it should not be slashable for miner A to mine on top of W. Mining on top of Z without including Y in their parent tipset is what we're targeting here, correct?

Reply:
should be fixed now, thanks for pointing this out